tests/test_orders.py

```diff
@@ -83,6 +83,12 @@ def test_get_order(client, response):
         order.payments
 
 
+def test_get_order_invalid_id(client):
+    with pytest.raises(IdentifierError) as excinfo:
+        client.orders.get("invalid")
+    assert str(excinfo.value) == "Invalid order ID 'invalid', it should start with 'ord_'."
```

Comment:
What happens if the order ID starts with `ord_` but the rest of the ID is invalid?

Reply:
The ID is just an identifier, invalid here seems to mean that you send `ord_invalid` or `ord_56789` in stead of `ord_12345`?

Then it;ll be sent to the API, and that will probably return a HTTP 404 error (depending which API operation you're doing). The client will then use that to raise a `RequestError`.

Reply:
Ok, gotcha.